www/shared/components/cookie-banner/CookieBanner.js

```diff
@@ -0,0 +1,42 @@
+import React, { useCallback } from 'react';
+import PropTypes from 'prop-types';
+import Link from 'next/link';
+import styles from './CookieBanner.module.css';
+import { useCookieConsents } from '@enzsft/react-cookie-consents';
+import { initializeTagManager } from '../../utils/google-tag-manager';
+
+export const CookieBanner = () => {
+    const cookieConsents = useCookieConsents();
+
+    const onAcceptCookies = useCallback(() => cookieConsents.add('analytics'), [cookieConsents]);
+
+    if (cookieConsents.get().length > 0) {
+        initializeTagManager();
+
+        return null;
+    }
+
+    return (
+        <>
```

Comment:
You are you using a Fragment tag when you can simply use the child span as the root?